test-loop-tests/src/utils/peer_manager_actor.rs

```diff
@@ -114,6 +153,33 @@ impl TestLoopPeerManagerActor {
         // over the handlers in reverse order.
         self.handlers.push(handler);
     }
+
+    fn push_network_info(
+        &self,
+        ctx: &mut dyn near_async::futures::DelayedActionRunner<Self>,
+        interval: Duration,
+    ) {
+        let future = self.client_sender.send_async(SetNetworkInfo(NetworkInfo {
+            highest_height_peers: self
+                .last_block_headers
+                .iter()
+                .map(|(peer_info, header)| HighestHeightPeerInfo {
+                    archival: false,
+                    genesis_id: self.genesis_id.clone(),
+                    highest_block_hash: *header.hash(),
+                    highest_block_height: header.height(),
+                    tracked_shards: vec![],
```

Comment:
If we are setting this info, might we also want to correctly populate `tracked_shards`? We could just rename and modify `TestLoopNetworkBlockInfo` to send the whole HighestHeightPeerInfo?

Also, a small comment on top of this function would help.

Reply:
Added a comment as part of 51474c9b64d2b2fe87063198747d17f4e83872af